src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -829,8 +829,20 @@ if (typeof Sfjs === 'undefined' || typeof Sfjs.loadToolbar === 'undefined') {
             initializeLogsTable: function() {
                 Sfjs.updateLogsTable();
 
-                document.querySelectorAll('.log-filter input').forEach((input) => {
-                    input.addEventListener('change', () => { Sfjs.updateLogsTable(); });
+                document.querySelectorAll('button[data-filter-name="type"]').forEach((tab) => {
+                    tab.addEventListener('click', () => {
+                        /* update the currently selected "log type" tab */
+                        document.querySelectorAll('button[data-filter-name="type"].tab-control').forEach((otherTab) => otherTab.classList.remove('active'));
+                        tab.classList.add('active');
```

Comment:
We need to handle tabs properly here, with full aria support (same than what `createTabs` does).

Alternatively, this could also keep using a radio group with a dedicated rendering making it look like `tab-navigation` but not being tabs (which might be better for accessibility as there is no tab panels in the log table so the aria tabs markup is still incomplete)